reference/7.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -310,16 +310,18 @@ PS Env:\> Get-ChildItem ComputerName
 
 ### Saving changes to environment variables
 
-To make a persistent change to an environment variable on Windows, use the
-System Control Panel. Select **Advanced System Settings**. On the **Advanced**
-tab, click **Environment Variable...**. You can add or edit existing
-environment variables in the **User** and **System** (Machine) scopes. Windows
-writes these values to the Registry so that they persist across sessions and
-system restarts.
+To make a persistent change to an environment variable on Windows, use either of
+the following:
+
+1. The [setx.exe](https://docs.microsoft.com/windows-server/administration/windows-commands/setx)
+   command-line utility.
+1. The Advanced tab of the System applet in Control Panel. Issuing the
+   `SystemPropertiesAdvanced.exe` command at the PowerShell prompt is a
+   straightforward way of accessing it.
```

Comment:
@skycommand Thanks for the contribution. I don't want to add the `set.exe` information. We already have information about using **System.Environment** methods for a command-line example. But we should fix the description of the Control Panel method since this has changed in Windows 10.

```suggestion
To make a persistent change to an environment variable on Windows 10, open the System settings using
<kbd>Win</kbd>+<kbd>I</kbd>. Type **View advanced system settings** in the search box. On the
Advanced tab, click **Environment Variable...**. You can add or edit existing environment variables
in the User and System (Machine) scopes. Windows writes these values to the Registry so that they
persist across sessions and system restarts.
```

Reply:
Too convoluted. Just press <kbd>Win</kbd>, type "path", and select "Edit environment variables for your account".